src/test/java/ro/tripleaunibuc/auth/TreeAProjectApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package ro.tripleaunibuc.auth;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class TreeAProjectApplicationTests {
```

Comment:
Nitpick: we should rename this to TripleAProjectApplicationTest or ThreeAProjectApplicationTests